### PR TITLE
Add workaround for network issues in migration scenarios

### PR DESCRIPTION
**Pull request by alvarocarvajald**
SLES+HA migration scenarios in openqa.suse.de are failing with apparent network timeouts and latency, affecting primarily DHCP and sbd; these network issues are not present in fresh installation scenarios.

This PR introduces in migration scenarios a restart of the `iscsi` and `pacemaker` services before checking the health of the cluster and its resources to give the cluster more time to finish network setup.

- Related ticket: N/A
- Needles: N/A
- Verification run: http://1b147.qa.suse.de/tests/5180
- Failing tests: https://openqa.suse.de/tests/2537853 (15GM, ppc64le), https://openqa.suse.de/tests/2539054 (12sp4, ppc64le), https://openqa.suse.de/tests/2538810 (12sp3, ppc64le), https://openqa.suse.de/tests/2535539 (15GM, aarch64), https://openqa.suse.de/tests/2539343 (15GM, x86_64), https://openqa.suse.de/tests/2538859 (12sp4, x86_64), https://openqa.suse.de/tests/2540033 (12sp3, x86_64)

### CODE REVIEW

File: tests/ha/check_after_reboot.pm [hunk 1]
```diff
@@ -36,6 +36,12 @@ sub run {
     # in that case
     select_console 'root-console' if (get_var('HDDVERSION'));
 
+    # Workaround network timeout issue during upgrade
+    if (get_var('HDDVERSION')) {
+        systemctl 'restart iscsi';
+        systemctl 'restart pacemaker';
```
Comment (ldevulder) [inline tests/ha/check_after_reboot.pm]: You forget to add `use utils 'systemctl';`.

Comment (alvarocarvajald) [reply tests/ha/check_after_reboot.pm]: Not anymore.

**Comment by juadk** [comment]
LGTM!

**Comment by ldevulder** [comment]
Forget to remove the WIP tag before merging it, sorry!

**Pull request merged**